logstash-core/src/main/java/org/logstash/plugins/PluginFactoryExt.java

```diff
@@ -74,6 +80,10 @@
 
         private RubyClass filterClass;
 
+        private ConfigVariableExpander cve;
```

Comment:
nitpick: let's have a more explanatory name when using the field - full name or smt like `variableExpander`